### PR TITLE
#3159843 - Styling for the date block on the create event page

### PR DESCRIPTION
**Pull request by agami4**
## Problem
Should update styles for the data&time block on the event-create page
Add new icons

## Solution
Update styles and add icons

## Issue tracker
https://www.drupal.org/project/social/issues/3159843

## How to test
*For example*
- [ ] Go to the event-create page.

## Screenshots
data time

## Release notes
The date&time block on the event-create page has a new style.
Added new icons: email and duplicate.

### CODE REVIEW

File: modules/social_features/social_event/social_event.module [hunk 1]
```diff
@@ -498,6 +498,14 @@ function social_event_block_access(Block $block, $operation, AccountInterface $a
  */
 function social_event_form_alter(&$form, FormStateInterface $form_state, $form_id) {
   social_event_show_pending_requests_message();
+
+  if (in_array($form_id, [
```
Comment (robertragas) [inline modules/social_features/social_event/social_event.module:502]: This should go in the switch below. I can help with that.

**Comment by bramtenhove** [comment]
@agami4 it looks really good, thanks.

There is now one issue which needs to be solved. The styling for the country and location is not there for group and profiles (or other places where it is used).

Can you make the styling for the country and location field generic for all of these?

**Styling at event**
Screenshot 2020-08-10 at 15 23 18

**Styling at group**
Screenshot 2020-08-10 at 15 23 30

**Styling at profile**
Screenshot 2020-08-10 at 15 23 37

File: modules/social_features/social_event/assets/css/social-event--admin.scss [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+              flex: 0 0 195px;
+              max-width: 190px;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .field--type-address {
```
Comment (bramtenhove) [inline modules/social_features/social_event/assets/css/social-event--admin.scss:58]: @agami4 can you put this in the global styling so it is applied on every form where the address field is used?

File: modules/social_features/social_event/social_event.libraries.yml [hunk 3]
```diff
@@ -15,3 +15,8 @@ modal:
     - core/jquery
     - core/drupal
     - core/drupalSettings
+
+admin:
+  css:
+    theme:
+      assets/css/social-event--admin.css: {}
```
Comment (bramtenhove) [inline modules/social_features/social_event/social_event.libraries.yml:22]: Can we rename this to `social-event--date.css`?

File: modules/social_features/social_event/social_event.module (hunk 1 above)
Comment (bramtenhove) [reply modules/social_features/social_event/social_event.module]: I merged the `if` and `switch` statements in 46090fa9e01362edc51e1335e9b998cc50c2671a

File: modules/social_features/social_event/assets/css/social-event--admin.scss (hunk 2 above)
Comment (agami4) [reply modules/social_features/social_event/assets/css/social-event--admin.scss]: I've moved general styles for location block to socialbase theme.

**Comment by ronaldtebrake** [comment]
Great job @agami4, thank you so much!

File: modules/social_features/social_event/src/Form/EnrollActionForm.php [hunk 4]
```diff
@@ -199,7 +199,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $attributes = [
       'class' => [
         'btn',
-        'btn-accent brand-bg-accent',
+        'btn-default brand-bg-default',
```
Comment (ronaldtebrake) [inline modules/social_features/social_event/src/Form/EnrollActionForm.php:202]: Just a FYI - i had to revert these changes as they gave issues in the default (no SKY) theme.
